rocAL/source/loaders/image/numpy_loader.cpp

```diff
@@ -154,8 +154,11 @@ NumpyLoader::load_routine() {
     LOG("Started the internal loader thread");
     LoaderModuleStatus last_load_status = LoaderModuleStatus::OK;
     // Initially record number of all the numpy arrays that are going to be loaded, this is used to know how many still there
-    auto max_shape = _output_tensor->info().max_shape();
-    auto num_dims = max_shape.size();
+    const std::vector<size_t> tensor_dims = _output_tensor->info().dims();
```

Comment:
You can replace all of the following new code into single line except line 159
std::vector<size_t> max_shape(tensor_dims.begin() + 1, tensor_dims.end());